From 2cb5ac9f7c68f3ae0bb8185f9355d169b66d5291 Mon Sep 17 00:00:00 2001
From: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Date: Tue, 8 Feb 2022 00:44:10 +0200
Subject: [PATCH 2/2] mk/gcc.mk: return back CXX${sm} definition

Renesas accidentally removed definition of CXX${} in the patch
"0001-mk-gcc.mk-Change-the-path-to-the-library.patch"

Remove back this definition as it is required for optee_test.

Signed-off-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
---
 mk/gcc.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mk/gcc.mk b/mk/gcc.mk
index e871699e..f89410d1 100644
--- a/mk/gcc.mk
+++ b/mk/gcc.mk
@@ -2,9 +2,11 @@
 ifdef PKG_CONFIG_SYSROOT_DIR
 CC$(sm)        := $(CROSS_COMPILE_$(sm))gcc --sysroot=$(PKG_CONFIG_SYSROOT_DIR)
 LD$(sm)        := $(CROSS_COMPILE_$(sm))ld.bfd --sysroot=$(PKG_CONFIG_SYSROOT_DIR)
+CXX$(sm)       := $(CROSS_COMPILE_$(sm))g++ --sysroot=$(PKG_CONFIG_SYSROOT_DIR)
 else
 CC$(sm)        := $(CROSS_COMPILE_$(sm))gcc
 LD$(sm)        := $(CROSS_COMPILE_$(sm))ld.bfd
+CXX$(sm)       := $(CROSS_COMPILE_$(sm))g++
 endif
 CPP$(sm)	:= $(CROSS_COMPILE_$(sm))cpp
 AR$(sm)		:= $(CROSS_COMPILE_$(sm))ar
-- 
2.35.1

